--- programs/elven/lv2host.cpp.orig	2010-01-20 15:37:41.417318302 +0100
+++ programs/elven/lv2host.cpp	2010-01-20 15:38:15.428147938 +0100
@@ -508,8 +508,8 @@
   if (!lv2p_c) {
     DBG1("LV2_PATH is not defined, will search default directories");
     search_dirs.push_back(string(getenv("HOME")) + "/.lv2");
-    search_dirs.push_back("/usr/lib/lv2");
-    search_dirs.push_back("/usr/local/lib/lv2");
+    search_dirs.push_back("/usr/lib64/lv2");
+    search_dirs.push_back("/usr/local/lib64/lv2");
   }
   else {
     string lv2_path = lv2p_c;
